bundles/config/org.eclipse.smarthome.config.xml/src/main/java/org/eclipse/smarthome/config/xml/osgi/XmlDocumentBundleTracker.java

```diff
@@ -58,7 +58,7 @@
  */
 public class XmlDocumentBundleTracker<T> extends BundleTracker<Bundle> {
 
-    public static final String THREAD_POOL_NAME = "xml-processing";
+    public static final String THREAD_POOL_NAME = "file-processing";
```

Comment:
You were probably already waiting for this question: Would it make sense to move this constant to some "common" place as it is declared twice now in different classes (but they are meant to be identical to each other)?